src/libos/src/net/socket/unix/addr.rs

```diff
@@ -10,7 +10,7 @@ lazy_static! {
 
 #[derive(Clone, Debug, Eq, PartialEq)]
 pub enum Addr {
-    File(UnixPath),
+    File(UnixPath, Option<usize>), // path or inode
```

Comment:
// path and an optional inode number

Reply:
Done.

Reply:
I am wondering why using a pair of `UnixPath` and `Option<usize>` instead of an `InodeFile`? Essentially, a pathname-based UDS should be backed by an inode. An inode provides more info than the combination of path and inode number.

Reply:
The reason is adding an optional parameter is an easy fix and will not affect existing code and function.